guides/common/modules/con_upgrading-smartproxyservers-in-a-load-balancing-setup.adoc

```diff
@@ -0,0 +1,7 @@
+[id="upgrading-smartproxyservers-in-a-load-balancing-setup_{context}"]
+= Upgrading {SmartProxyServers} in a load balancing setup
+
+ifdef::satellite[]
+To upgrade {SmartProxyServers} from {ProjectVersionPrevious} to {ProjectVersion}, complete the {UpgradingDocURL}upgrading_capsule_server[Upgrading {SmartProxyServers}] procedure in _{UpgradingDocTitle}_.
```

Comment:
I updated the link for Satellite.

I'm not adding the link for Katello (unless someone finds it really important to add it -- let me know!). To be honest, the whole module feels like it may not even be needed. But it was here before and removing it or changing it significantly is out of scope right now.